src/main/java/bio/terra/stairway/Stairway.java

```diff
@@ -233,6 +240,45 @@ public Builder workQueueSubscriptionId(String workQueueSubscriptionId) {
       return this;
     }
 
+    /**
+     * Control the frequency of clean up passes over the retained flights. Defaults to never
+     * checking.
```

Comment:
Defaulting to checking once an hour or day seems better. This can still not do anything if no retention is set.

Reply:
Fair enough.